main/src/main/java/org/mobilitydata/gtfsvalidator/util/HtmlOutputUtil.java

```diff
@@ -0,0 +1,199 @@
+  public static final String STYLE_CSS = "style.css";
+  public static final String SCRIPT_JS = "script.js";
+  private static final String ERROR = SeverityLevel.ERROR.name();
+  private static final String WARNING = SeverityLevel.WARNING.name();
+  private static final String INFO = SeverityLevel.INFO.name();
+  private static final String NOTICES = "notices";
+  private static final String CODE = "code";
+  private static final String SEVERITY = "severity";
+  private static final String TOTAL_NOTICES = "totalNotices";
+  private static final String SAMPLE_NOTICES = "sampleNotices";
+
+  public static String outputBuilder(
+      JsonObject noticesJson, String jqueryDependency, String styleCss, String scriptJs) {
+    StringBuilder notices = new StringBuilder();
+    HashMap<String, Integer> noticesQty =
+        new HashMap<>() {
+          {
+            put(ERROR, 0);
+            put(WARNING, 0);
+            put(INFO, 0);
+          }
+        };
+    JsonArray noticesArray = noticesJson.getAsJsonArray(NOTICES);
+    for (JsonElement notice : noticesArray) {
+      JsonObject noticeJson = notice.getAsJsonObject();
+      String noticeCode = noticeJson.get(CODE).getAsString();
+      String noticeSeverity = noticeJson.get(SEVERITY).getAsString();
+      int totalNotices = noticeJson.get(TOTAL_NOTICES).getAsInt();
+      noticesQty.put(noticeSeverity, noticesQty.get(noticeSeverity) + totalNotices);
+      JsonArray sampleNotices = noticeJson.getAsJsonArray(SAMPLE_NOTICES);
+      JsonElement sampleNotice = sampleNotices.get(sampleNotices.size() - 1);
+      JsonObject sampleJson = sampleNotice.getAsJsonObject();
+      ArrayList<String> noticeFields = new ArrayList<>();
+      Set<Map.Entry<String, JsonElement>> sampleEntries = sampleJson.entrySet();
+      for (Map.Entry<String, JsonElement> entry : sampleEntries) {
+        noticeFields.add(entry.getKey());
+      }
+      String sampleColumns = noticeColumnsBuilder(noticeFields);
+      String sampleRows = noticeRowsBuilder(noticeFields, sampleNotices);
+      notices.append(
+          noticeFrame(noticeCode, noticeSeverity, totalNotices, sampleColumns, sampleRows));
+    }
+    String outputHeader =
+        outputHeader(
+            (noticesQty.get(ERROR) + noticesQty.get(WARNING) + noticesQty.get(INFO)),
+            noticesQty.get(ERROR),
+            noticesQty.get(WARNING),
+            noticesQty.get(INFO));
+    return outputFrame(
+        jqueryDependency, styleCss, scriptJs, outputHeader, notices.toString(), outputFooter());
+  }
+
+  public static String resourceLoader(String resourcePath) {
+    String content = "";
+    try (InputStream stream =
+        HtmlOutputUtil.class.getClassLoader().getResourceAsStream(resourcePath)) {
+      Scanner scanner = new Scanner(stream).useDelimiter("\\A");
+      content = scanner.hasNext() ? scanner.next() : "";
+    } catch (IOException e) {
+      System.out.println("The resource was not found at \"" + resourcePath + "\".");
```

Comment:
Also, I think the main result of this would be the HTML file would be malformed/broken? If so you should probably add that to the output message, otherwise the user might not tie these two things together.